systemtest/src/test/java/io/strimzi/systemtest/BaseST.java

```diff
@@ -510,7 +510,7 @@ protected void checkReadinessLivenessProbe(String podNamePrefix, String containe
         }
     }
 
-    void verifyLabelsForKafkaCluster(String clusterName, String appName) {
+    protected void verifyLabelsForKafkaCluster(String clusterName, String appName) {
         verifyLabelsForKafkaOrZkPods(clusterName, "zookeeper", appName);
```

Comment:
Probably best to split this function into two different ones, i.e. 
verifyLabelsForKafkaPods(clusterName, appName);
verifyLabelsForZookeeperPods(clusterName, appName);

Reply:
Maybe we could use `verifyLabelsOnPods` and remove `verifyLabelsForKafkaOrZkPods`completely, because its do bsically the same thing